src/main/java/org/jenkinsci/plugins/ghprb/GhprbRootAction.java

```diff
@@ -84,79 +104,172 @@ public void doIndex(StaplerRequest req, StaplerResponse resp) {
 
-            if ("issue_comment".equals(event)) {
-                IssueComment issueComment = getIssueComment(payload, gh);
-                GHIssueState state = issueComment.getIssue().getState();
+            if (StringUtils.equalsIgnoreCase("issue_comment", event)) {
+
+                comment = getIssueComment(payload, gh);
+                GHIssueState state = comment.getIssue().getState();
+
                 if (state == GHIssueState.CLOSED) {
                     logger.log(Level.INFO, "Skip comment on closed PR");
                     return;
                 }
-                
-                if (!issueComment.getIssue().isPullRequest()) {
+
+                if (!comment.getIssue().isPullRequest()) {
                     logger.log(Level.INFO, "Skip comment on Issue");
                     return;
                 }
 
-                String repoName = issueComment.getRepository().getFullName();
+                repoName = comment.getRepository().getFullName();
 
-                logger.log(Level.INFO, "Checking issue comment ''{0}'' for repo {1}", new Object[] { issueComment.getComment().getBody(), repoName });
+                logger.log(Level.INFO,
+                           "Checking issue comment ''{0}'' for repo {1}",
+                           new Object[] { comment.getComment().getBody(), repoName });
 
-                for (GhprbTrigger trigger : getTriggers(repoName, body, signature)) {
-                    try {
-                        IssueComment authedComment = getIssueComment(payload, trigger.getGitHub());
-                        trigger.handleComment(authedComment);
-                    } catch (Exception e) {
-                        logger.log(Level.SEVERE, "Unable to process web hook for: " + trigger.getProjectName(), e);
-                    }
-                }
+            } else if (StringUtils.equalsIgnoreCase("pull_request", event)) {
 
-            } else if ("pull_request".equals(event)) {
-                PullRequest pr = getPullRequest(payload, gh);
-                String repoName = pr.getRepository().getFullName();
+                pr = getPullRequest(payload, gh);
+                repoName = pr.getRepository().getFullName();
 
                 logger.log(Level.INFO, "Checking PR #{1} for {0}", new Object[] { repoName, pr.getNumber() });
 
-                for (GhprbTrigger trigger : getTriggers(repoName, body, signature)) {
-                    try {
-                        PullRequest authedPr = getPullRequest(payload, trigger.getGitHub());
-                        trigger.handlePR(authedPr);
-                    } catch (Exception e) {
-                        logger.log(Level.SEVERE, "Unable to process web hook for: " + trigger.getProjectName(), e);
-                    }
-                }
             } else {
-                logger.log(Level.WARNING, "Request not known");
+                logger.log(Level.WARNING, "Request not known for event: {0}", new Object[] { event });
```

Comment:
I would bump this down to a FINE or FINER.  There are other events under the pull request heading which aren't interesting that fall under this path.